content/en/xlt/load-testing/Manual/570-scorecard.md

```diff
@@ -9,7 +9,210 @@ description: >
+
+```json
+{
+    "selectors" : [
+        {
+            "id": "agentsWithCPULarger60p", // the ID of the selector - required
+            "expression": "count(//agents//totalCpuUsage/mean[number() > 60])", // the XPath expression of the selector - required
+            "comment" : "I can write a note to myself here"  // some comment; won't be evaluated - optional
+        }
+    ]
+}
+```
+
+You can then use the selector in any rule check by referencing its selector id:
+
+```json
+"checks" : [
+    {
+        "selectorId": "agentsWithCPULarger60p", //predefined selector
+        "condition": "= 0"
+    }
+]
+```
+
+### Groups
+
+To model more complex criteria and to allow re-use of rules (and thus to avoid repetiitve definitions), rules are assigned to groups. A rule can be assigned to multiple groups. Rules must be assigned to a least one (enabled) group to become effective.
+
+At least one enabled group having at least one enabled rule assigned must be specified in `groups` as follows:
+
+```json
+ "groups":[
+    {
+        "id": "homepageGroup", // the ID of the group for reference and listing in the report - required and unique
+        "name": "Homepage", // the (more or less descriptive) name of the group - optional
+        "comment": "Lorem ipsum ...", // some comment; won't be evaluated - optional
+        "enabled": true, // helps to disable the entire group - optional, default = true
+        "failsTest": false, // when true will mark the test as failed when the group failed - optional, default = false
+        "mode": "allPassed", // the group's mode; used to define the source of points as well as the outcome of the group state; must be one of "firstPassed", "lastPassed" or "allPassed - optional, default = 'firstPassed'
+        "description": "Just any text that explains the purpose", //  the text to display in the report, kinda help text like - optional, default empty
+        // the list of rules assigned to this group and to evaluate in that order - required, at least one rule per group
+        "rules": ["homepageResponseTimeA", "homepageResponseTimeB", "homepageResponseTimeC"]
+    }
+]
+```
+
+Groups are evaluated in the order of definition. Each rule evaluation triggers the evaluation of all rules assigned to it, in order of assignment. Rules that do not pass won't stop the evaluation of the remaining rules.
+
+The number of achievable points as well as the computed number of achieved points depend on the group's **mode**:
+
+| Mode | Achievable Points | Achieved Points |
+|--------|--------|--------|
+| _firstPassed_ | Max. of all rules' achievable points | First rule that passes |
+| _lastPassed_ | Max. of all rules' achievable points | Last rule that passes  |
+| _allPassed_ | Sum of all rules' achievable points | Sum of all rules that pass  |
+
+
+### Rating
+
+You can define a final rating for the entire test including the ability to mark it failed, following this pattern:
+
+```json
```

Comment:
JSON does not support comments. Either we have to use js as language for highlighting or remove the comments.

```suggestion
```js
```